### ISSUE TITLE
How to bootstrap development environment?

### ISSUE THREAD

**Issue by couchand** [stm32-rs/svdtools#40]
I pulled down the repository and tried to run make.  After setting up python venv, I'm getting:

```
make: venv/bin/pip: Command not found
```

Am I missing some critical first step here?  Forgive me, I've never used python venv before.

**Comment by nickray** [comment]
Does `make update-venv` help?

**Comment by couchand** [comment]
Nope, that's where I hit the command not found error: https://github.com/stm32-rs/svdtools/blob/145d5e5f9c0fbee767bdca1f4d27da3265e6f73c/Makefile#L59
